### PR TITLE
Improvements/add light theme

### PR DESCRIPTION
**Pull request by helloimela**
**Describe pull-request**  
Add light theme with css vars.
How css vars work:
- Pay attention to changes under theme/light/src/styles/core
- gulpfile.babel.js L268 & function on L346 auto-generate css-vars file
- Checkout in components/header, components/dealer-header, components/footer, all use css variables and adoptedStyle is removed

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: https://github.com/scania/scania-theme/issues/159

**How to test**  
1. Clone sdds repo
2. build and install components and theme/light
3. run storybook inside components (npm run sb)
4. Look at the header & dealer header

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._

### CODE REVIEW

**Comment by helloimela** [comment]
You can also check package installed on the example demo page in angular https://github.com/scania-digital-design-system/sdds/pull/5

File: theme/light/gulpfile.babel.js [hunk 1]
```diff
@@ -337,6 +343,25 @@ document.addEventListener('storeReady', function(event) {
   cb();
 }
 
+function generateVars(input){
```
Comment (mmexvr) [inline theme/light/gulpfile.babel.js:346]: **In the future** maybe we should seperate all this functions into a bundler pack or something. Keeping the file a bit smaller.

File: theme/light/src/styles/core/typography/typography-vars.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+--font-weight-base: var(--font-weight-normal);
+--line-height-base:            1.5;
+
+--font-family-sans-serif:      'Scania Sans Semi Condensed', 'Scania Sans Condensed', Arial, Helvetica, sans-serif;
+--font-family-sans:            'Scania Sans', Arial, Helvetica, sans-serif;
+--font-family-headline:        'Scania Sans Headline', Arial, Helvetica, sans-serif;
+--font-family-monospace: var(--font-family-sans-serif);
+--font-family-base: var(--font-family-sans-serif);
+
+--h1-font-size: var(--font-size-base * 1.72);
```
Comment (mmexvr) [inline theme/light/src/styles/core/typography/typography-vars.scss:21]: This doesn't work, but it is getting fixed in typograhpy PR, will leave it for now

Comment (mmexvr) [reply theme/light/src/styles/core/typography/typography-vars.scss:21]: It seems it doesn't get value, but a string. Good to keep it in mind for the future

Comment (mmexvr) [reply theme/light/src/styles/core/typography/typography-vars.scss:21]: If you move * 1.72 outside it works

**Comment by mmexvr** [comment]
Looks good!

**Pull request merged**